Project.toml

```
@@ -25,6 +26,7 @@ TrixiBase = "9a0f1c46-06d5-4909-a5a3-ce25d3fa3284"
 [compat]
 BandedMatrices = "1"
 DiffEqBase = "6.143"
+FastBroadcast = "0.3"
```

```suggestion
FastBroadcast = "0.2.8, 0.3"
```
As explained above we need at most v3.3 of RecursiceArrayTools.jl for Julia v1.9, but RecursiceArrayTools.jl v3.3 [only supports FastBroadcast <= 0.2.8](https://github.com/SciML/RecursiveArrayTools.jl/blob/v3.3.3/Project.toml#L40). So we would also need to support FastBroadcast.jl v0.2.8, right?